src/commands/appSettings/AppSettingsClient.ts

```diff
@@ -16,25 +16,32 @@ export class AppSettingsClient implements IAppSettingsClient {
     public fullName: string;
     public resourceGroup: string;
     public root: ISubscriptionContext;
+    public prId: string;
+    public isBuild: boolean;
 
     constructor(node: EnvironmentTreeItem) {
         this.ssId = node.id;
         this.fullName = node.parent.name;
         this.resourceGroup = node.parent.resourceGroup;
         this.root = node.root;
 
+        this.prId = node.buildId;
```

Comment:
Why not call this `buildId` instead?

Reply:
I chose `prId` because that's the name of the property in the API, but I'd be fine calling it `buildId` as well.
![image](https://user-images.githubusercontent.com/5290572/116915960-8cac9f00-ac01-11eb-82ae-f8c24a8fa2e6.png)